runtime/parser2/expression.go

```diff
@@ -561,6 +562,21 @@ func defineReferenceExpression() {
 	)
 }
 
+func defineMemberExpression() {
+	setExprLeftBindingPower(lexer.TokenDot, 140)
```

Comment:
Maybe bump this to the same precedence as indexing and function calls:

```suggestion
	setExprLeftBindingPower(lexer.TokenDot, 150)
```